src/wasm-traversal.h

```diff
@@ -50,6 +50,7 @@ struct Visitor {
   ReturnType visitLoad(Load* curr) {}
   ReturnType visitStore(Store* curr) {}
   ReturnType visitAtomicRMW(AtomicRMW* curr) {return ReturnType();} //Stub impl so not every pass has to implement this yet.
```

Comment:
I have a local branch with that change which I'll submit as a separate PR because it depends on this one (I already had this branch in flight when we decided that).